apps/tunlookup/tunlookup.yml

```diff
@@ -0,0 +1,10 @@
+name: Tunlookup
+description: 'Translate selected text with a keystroke'
+website: 'https://tun-application.github.io/'
+keywords:
+    - Tunlookup
+    - Translate selected text with a keystroke
```

Comment:
These first two keywords can be removed, as they are the same as the name and description of the app.